src/routes/privacy-policy/+page.svelte

```diff
@@ -0,0 +1,84 @@
+<script lang="ts">
+	import { pageTitle } from '../../stores.js';
+	pageTitle.set('Privacy Policy');
+</script>
+
+<div class="secondary-bg">
+	<div class="section">
+		<h1>Privacy Policy</h1>
+		<p class="emphasis">
+			This privacy notice tells you what to expect us to do with your personal information.
+		</p>
+
+		<ul>
+			<li><a href="#contact-details">Contact details</a></li>
+			<li><a href="#what-information">What information we collect, use, and why</a></li>
+			<li><a href="#lawful-bases">Lawful bases and data protection rights</a></li>
+			<li><a href="#personal-information-source">Where we get personal information from</a></li>
+			<li><a href="#retention">How long we keep information</a></li>
+			<li><a href="#complaints">How to complain</a></li>
+		</ul>
+
+		<h2 id="contact-details">Contact details</h2>
+		<p>Email: <a href="mailto:management@dddsouthwest.com">management@dddsouthwest.com</a></p>
+
+		<h2 id="what-information">What information we collect, use, and why</h2>
+		<p>We collect or use the following personal information to provide updates and share news:</p>
+		<ul>
+			<li>Names</li>
+			<li>Email address</li>
```

Comment:
When signing up for a ticket last year, we also collected information about gender (for anonymised metrics) and allergies and access needs (for supporting attendees on the day)